e2e/atlas/live_migrations_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && livemigrations)
+// +build e2e atlas,livemigrations
+
+package atlas_test
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongocli/e2e"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLinkToken(t *testing.T) {
+	cliPath, err := e2e.Bin()
+	r := require.New(t)
+	r.NoError(err)
+
+	// Cleanup, do a delete in case a token already exists from another run of the test
+	log.Printf("Removing link-tokens from previous/parallel test runs...")
```

Comment:
[nit] you can use the test log https://pkg.go.dev/testing#T.Log
```suggestion
	t.Logf("Removing link-tokens from previous/parallel test runs...\n")
```

Reply:
Thanks, hunted around for a info/warn/error log handler first, didn't think of Log 🤦